invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/app.js

```diff
@@ -6,9 +6,7 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import { initSearchApp } from ".";
+import defaultComponents from "./defaultComponents";
+import { createSearchAppInit } from "./util";
 
-const searchAppElements= document.querySelectorAll("[data-invenio-search-config]");
-for (const appRootElement of searchAppElements) {
-  initSearchApp(appRootElement);
-}
+export const initSearchApp = createSearchAppInit( defaultComponents, true );
```

Comment:
**Minor:** I think you don't need to pass `true` as it's the default.

Question: Why is it needed here to export `initSearchApp`? Aren't you already initializing the apps when `autoInit = true`?